### PR TITLE
AAP-28434 Added notes about the impact of Redis failure on EDA controller resources

### PR DESCRIPTION
**Pull request by jself-sudoku**
Added notes in 3 places about the impact of HA Redis failure on EDA controller resources:

- downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc
- downstream/assemblies/eda/assembly-eda-projects.adoc
- downstream/assemblies/eda/assembly-eda-rulebook-activations.adoc

### CODE REVIEW

File: downstream/assemblies/eda/assembly-eda-projects.adoc [hunk 1]
```diff
@@ -6,6 +6,12 @@ Projects are a logical collection of rulebooks.
 They must be a git repository and only http protocol is supported. 
 The rulebooks of a project must be located in the path defined for {EDAName} content in Ansible collections: `/extensions/eda/rulebooks` at the root of the project.
 
+[NOTE]
+====
+* Do not link a project to an empty repository. Linking a project to an empty repository causes synchronization errors.
```
Comment (jshimkus-rh) [inline downstream/assemblies/eda/assembly-eda-projects.adoc:11]: I believe this is no longer the case.  See: https://github.com/ansible/eda-server/pull/988

File: downstream/assemblies/eda/assembly-eda-projects.adoc [hunk 2]
```diff
@@ -6,6 +6,12 @@ Projects are a logical collection of rulebooks.
...
+* When Redis is unavailable due to a failure, you will not be able to create or sync projects.
```
Comment (jshimkus-rh) [inline downstream/assemblies/eda/assembly-eda-projects.adoc:12]: I'm not certain about the inclusion of "due to a failure."  It's fairly generic and doesn't really add anything, imo.
Perhaps something along the line of "If EDA cannot successfully communicate with Redis..." ?  But I might be acting too "engineery".

File: downstream/assemblies/eda/assembly-eda-rulebook-activations.adoc [hunk 3]
```diff
@@ -6,6 +6,17 @@
 
 A rulebook activation is a process running in the background defined by a decision environment executing a specific rulebook.
 
+[IMPORTANT]
+====
+To meet high availability demands, {EDAcontroller} shares centralized link:https://redis.io/[Redis (REmote DIctionary Server)] with the {PlatformNameShort} UI. When Redis is unavailable due to failure, the following functions will not be available:
```
Comment (jshimkus-rh) [inline downstream/assemblies/eda/assembly-eda-rulebook-activations.adoc:11]: Similar comment re "unavailable due to failure."

File: downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc [hunk 4]
```diff
@@ -17,7 +17,6 @@ The following procedures form the user configuration:
 [NOTE]
 
 ====
-
-API documentation for {EDAcontroller} is available at \https://<eda-server-host>/api/eda/v1/docs
-
+* API documentation for {EDAcontroller} is available at \https://<eda-server-host>/api/eda/v1/docs
+* To meet high availability demands, {EDAcontroller} shares centralized link:https://redis.io/[Redis (REmote DIctionary Server)] with the {PlatformNameShort} UI. When Redis is unavailable due to a failure, you will not be able to create projects or enable rulebook activations.
```
Comment (jshimkus-rh) [inline downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc:21]: Similar comment re "unavailable due to failure."

File: downstream/assemblies/eda/assembly-eda-projects.adoc (hunk 2 above)
Comment (jself-sudoku) [reply downstream/assemblies/eda/assembly-eda-projects.adoc:12]: I can see your point with that @jshimkus-rh. I followed some suggestions, but will remove that portion. Thanks for feedback on that.

File: downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc (hunk 4 above)
Comment (jself-sudoku) [reply downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc:21]: Got it. Will remove.

File: downstream/assemblies/eda/assembly-eda-projects.adoc (hunk 1 above)
Comment (jself-sudoku) [reply downstream/assemblies/eda/assembly-eda-projects.adoc:11]: That note was added separately for another jira. I can look into this and make sure it gets removed if that's the case.

**Review by jshimkus-rh — APPROVED** [review]

**Review by hherbly — APPROVED** [review]
LGTM

**Pull request merged**